README.md

```diff
@@ -240,6 +241,35 @@ jobs:
 
 Though this is also possible to do with GitHub Actions, click [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on) for more information
 
+### Deploy custom remote branch
+
+You can use the **remote_branch** option to deploy an app to another remote branch
+
+_.github/workflows/main.yml_
+
+```yaml
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master # Changing the branch here would also work
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.8.9 # This is the action
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: "YOUR APP's NAME" #Must be unique in Heroku
+          heroku_email: "YOUR EMAIL"
+          branch: "main"
+```
+
```

Comment:
Hey could you provide an example using the `remote_branch` property for reference. The current one seems to be using `branch` which I think is a typo

Reply:
sure.